app/views/layouts/events/_what_you_will_learn.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="row">
+  <div class="container-fluid">
+    <div class="container">
+      <h1>What You Will Learn</h1>
+      <h3>Professional developers technologies</h3>
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-4">
```

Comment:
I think we might want these to be col-md-4 rather than col-lg-4. Once you are on a browser window smaller than 1200px, I think this will all stack to single column.